### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by danwie** [CocoCR300/flauncher#13]
I downloaded and sideloaded the app onto my TCL Google TV, but when I try to install it asks me if I want to update the original flauncher to your app. I choose yes and then get a message saying that it could not install. I've tried the universal apk. Do I need to uninstall the original flauncher first (I will then loose all my settings?)

Another question (I've only tried this with the original flauncher): I have installed Termux on my TV and connected using bluetooth keyboard. I've then typed the two commands "adb shell pm disable-user....". But I get the error "No devices/emulators found". Not sure what I'm doing wrong.

**Comment by hagst** [comment]
> Do I need to uninstall the original flauncher first (I will then loose all my settings?)
> 
Yes you need to uninstall the original flauncher because both apps use the same package name.
Yes you will loose all your settings.

> Another question (I've only tried this with the original flauncher): I have installed Termux on my TV and connected using bluetooth keyboard. I've then typed the two commands "adb shell pm disable-user....". But I get the error "No devices/emulators found". Not sure what I'm doing wrong.

I use [Launcher Manager (Android TV)](https://forum.xda-developers.com/t/app-firetv-noroot-launcher-manager-change-launcher-without-root.4176349/) for disabling the default launcher and select the launcher i want to use. With this i can switch between the default launcher, flauncher, [alauncher](https://github.com/4v3ngR/aLauncher) and other launchers.

**Comment by danwie** [comment]
> I use [Launcher Manager (Android TV)](https://forum.xda-developers.com/t/app-firetv-noroot-launcher-manager-change-launcher-without-root.4176349/) for disabling the default launcher and select the launcher i want to use. With this i can switch between the default launcher, flauncher, [alauncher](https://github.com/4v3ngR/aLauncher) and other launchers.

When restarting the device, will Launcher Manager be able to start the custom launcher automatically?

**Comment by hagst** [comment]
Yes
